imagetool/src/main/java/com/oracle/weblogic/imagetool/cachestore/CacheStoreFactory.java

```diff
@@ -17,10 +17,6 @@
         cashStoreMap.put(Constants.FILE_CACHE, FileCacheStore.CACHE_STORE);
     }
 
-    public CacheStore getCacheStore(String backingType) {
-        return cashStoreMap.getOrDefault(backingType.toUpperCase(), FileCacheStore.CACHE_STORE);
-    }
-
```

Comment:
"cash" lol

Reply:
I didn't write it :), I just deleted it.